--- libgpod-0.4.0/bindings/python/Makefile.am.orig	2006-05-30 15:21:09.000000000 +0200
+++ libgpod-0.4.0/bindings/python/Makefile.am	2006-11-15 11:02:49.439505000 +0100
@@ -32,14 +32,14 @@
 	$(CC) $(MODULE_LIBS) @PYTHON_LDFLAGS@ gpod_wrap.o -o $@
 
 install-pythonDATA: $(python_DATA)
-	$(mkinstalldirs) $(DESTDIR)$(pythondir)/gpod
-	$(INSTALL_PROGRAM) _gpod.so  $(DESTDIR)$(pythondir)/gpod/_gpod.so
-	$(INSTALL_DATA) *.py         $(DESTDIR)$(pythondir)/gpod/
-	$(PYTHON) -c 'from compileall import compile_dir; compile_dir("$(DESTDIR)$(pythondir)/gpod/", ddir="$(pythondir)/gpod/")'
-	$(PYTHON) -O -c 'from compileall import compile_dir; compile_dir("$(DESTDIR)$(pythondir)/gpod/", ddir="$(pythondir)/gpod/")'
+	$(mkinstalldirs) $(DESTDIR)$(pyexecdir)/gpod
+	$(INSTALL_PROGRAM) _gpod.so  $(DESTDIR)$(pyexecdir)/gpod/_gpod.so
+	$(INSTALL_DATA) *.py         $(DESTDIR)$(pyexecdir)/gpod/
+	$(PYTHON) -c 'from compileall import compile_dir; compile_dir("$(DESTDIR)$(pyexecdir)/gpod/", ddir="$(pyexecdir)/gpod/")'
+	$(PYTHON) -O -c 'from compileall import compile_dir; compile_dir("$(DESTDIR)$(pyexecdir)/gpod/", ddir="$(pyexecdir)/gpod/")'
 
 uninstall-pythonDATA: $(python_DATA)
-	rm -rf $(DESTDIR)$(pythondir)/gpod
+	rm -rf $(DESTDIR)$(pyexecdir)/gpod
 
 test:
 	cd tests && python tests.py
